src/user/index.tsx

```diff
@@ -14,15 +15,22 @@ interface IUserInfo {
   coverImage: string;
 }
 
+interface IFollowList {
+  followers: string[];
+  following: string[];
+}
+
 const COVER_IMG_URL = 'https://ifh.cc/g/xBfBwB.png';
-const PROFIE_IMG_URL = 'https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_1280.png';
+const PROFIE_IMG_URL = 'https://ifh.cc/g/35RDD6.png';
 const LIKE_IMG_URL = 'https://ifh.cc/g/vmscWK.png';
 const API_URL = 'http://kdt.frontend.3rd.programmers.co.kr:5006';
 
 const User = () => {
   const { id } = useParams();
```

Comment:
p4; 
현재 방문한 페이지에 따라 다른 결과를 redner 하므로 변수명을 수정해도 좋을 것 같습니다!
```suggestion
  const { id:  currentPageId } = useParams();
```